api/types.go

```diff
@@ -8,9 +8,10 @@ import (
 	datatransfer "github.com/filecoin-project/go-data-transfer"
 	"github.com/filecoin-project/go-state-types/abi"
 	"github.com/filecoin-project/lotus/build"
-	"github.com/filecoin-project/specs-actors/actors/builtin/miner"
 	"github.com/ipfs/go-cid"
 
+	v0miner "github.com/filecoin-project/specs-actors/actors/builtin/miner"
```

Comment:
Just FYI, the actor code uses a pattern like `miner0`. It's not necessary to match though.

Reply:
I don't really mind either way. But the trailing number is kind of nice. I'll refactor.